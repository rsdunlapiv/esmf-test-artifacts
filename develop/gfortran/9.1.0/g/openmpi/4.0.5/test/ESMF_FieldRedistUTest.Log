2022-07-14 01:49:41
20220714 020811.282 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 020811.282 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 020811.282 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 020811.282 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 020811.282 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 020811.282 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 020811.282 INFO             PET0 Running with ESMF Version   : v8.4.0b04
20220714 020811.282 INFO             PET0 ESMF library build date/time: "Jul 14 2022" "01:46:36"
20220714 020811.282 INFO             PET0 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_9.1.0_openmpi_g_develop/esmf
20220714 020811.282 INFO             PET0 ESMF_COMM                   : openmpi
20220714 020811.282 INFO             PET0 ESMF_MOAB                   : enabled
20220714 020811.282 INFO             PET0 ESMF_LAPACK                 : enabled
20220714 020811.282 INFO             PET0 ESMF_NETCDF                 : enabled
20220714 020811.282 INFO             PET0 ESMF_PNETCDF                : disabled
20220714 020811.282 INFO             PET0 ESMF_PIO                    : enabled
20220714 020811.282 INFO             PET0 ESMF_YAMLCPP                : enabled
20220714 020811.283 INFO             PET0  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20220714 020811.283 INFO             PET0  NUMBER_OF_PROCESSORS           4
20220714 020811.285 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.287 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.287 INFO             PET0  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20220714 020811.288 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.289 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.290 INFO             PET0  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20220714 020811.291 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.294 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.294 INFO             PET0  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20220714 020811.305 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.319 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.319 INFO             PET0  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20220714 020811.329 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.345 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.346 INFO             PET0  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20220714 020811.350 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.351 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.352 INFO             PET0  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20220714 020811.352 INFO             PET0  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20220714 020811.352 INFO             PET0 Finalizing ESMF
20220714 020811.282 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 020811.282 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 020811.282 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 020811.282 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 020811.282 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 020811.282 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 020811.282 INFO             PET1 Running with ESMF Version   : v8.4.0b04
20220714 020811.282 INFO             PET1 ESMF library build date/time: "Jul 14 2022" "01:46:36"
20220714 020811.282 INFO             PET1 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_9.1.0_openmpi_g_develop/esmf
20220714 020811.282 INFO             PET1 ESMF_COMM                   : openmpi
20220714 020811.282 INFO             PET1 ESMF_MOAB                   : enabled
20220714 020811.282 INFO             PET1 ESMF_LAPACK                 : enabled
20220714 020811.282 INFO             PET1 ESMF_NETCDF                 : enabled
20220714 020811.282 INFO             PET1 ESMF_PNETCDF                : disabled
20220714 020811.282 INFO             PET1 ESMF_PIO                    : enabled
20220714 020811.282 INFO             PET1 ESMF_YAMLCPP                : enabled
20220714 020811.283 INFO             PET1  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20220714 020811.283 INFO             PET1  NUMBER_OF_PROCESSORS           4
20220714 020811.285 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.287 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.287 INFO             PET1  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20220714 020811.288 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.289 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.290 INFO             PET1  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20220714 020811.291 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.294 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.294 INFO             PET1  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20220714 020811.305 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.319 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.319 INFO             PET1  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20220714 020811.329 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.345 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.346 INFO             PET1  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20220714 020811.350 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.351 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.352 INFO             PET1  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20220714 020811.352 INFO             PET1  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20220714 020811.352 INFO             PET1 Finalizing ESMF
20220714 020811.281 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 020811.281 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 020811.281 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 020811.281 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 020811.281 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 020811.282 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 020811.282 INFO             PET2 Running with ESMF Version   : v8.4.0b04
20220714 020811.282 INFO             PET2 ESMF library build date/time: "Jul 14 2022" "01:46:36"
20220714 020811.282 INFO             PET2 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_9.1.0_openmpi_g_develop/esmf
20220714 020811.282 INFO             PET2 ESMF_COMM                   : openmpi
20220714 020811.282 INFO             PET2 ESMF_MOAB                   : enabled
20220714 020811.282 INFO             PET2 ESMF_LAPACK                 : enabled
20220714 020811.282 INFO             PET2 ESMF_NETCDF                 : enabled
20220714 020811.282 INFO             PET2 ESMF_PNETCDF                : disabled
20220714 020811.282 INFO             PET2 ESMF_PIO                    : enabled
20220714 020811.282 INFO             PET2 ESMF_YAMLCPP                : enabled
20220714 020811.283 INFO             PET2  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20220714 020811.283 INFO             PET2  NUMBER_OF_PROCESSORS           4
20220714 020811.285 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.287 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.287 INFO             PET2  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20220714 020811.288 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.289 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.290 INFO             PET2  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20220714 020811.291 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.294 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.294 INFO             PET2  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20220714 020811.304 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.319 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.319 INFO             PET2  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20220714 020811.328 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.345 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.346 INFO             PET2  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20220714 020811.350 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.351 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.352 INFO             PET2  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20220714 020811.352 INFO             PET2  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20220714 020811.352 INFO             PET2 Finalizing ESMF
20220714 020811.282 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 020811.282 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 020811.282 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 020811.282 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 020811.282 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 020811.282 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 020811.282 INFO             PET3 Running with ESMF Version   : v8.4.0b04
20220714 020811.282 INFO             PET3 ESMF library build date/time: "Jul 14 2022" "01:46:36"
20220714 020811.282 INFO             PET3 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_9.1.0_openmpi_g_develop/esmf
20220714 020811.282 INFO             PET3 ESMF_COMM                   : openmpi
20220714 020811.282 INFO             PET3 ESMF_MOAB                   : enabled
20220714 020811.282 INFO             PET3 ESMF_LAPACK                 : enabled
20220714 020811.282 INFO             PET3 ESMF_NETCDF                 : enabled
20220714 020811.282 INFO             PET3 ESMF_PNETCDF                : disabled
20220714 020811.282 INFO             PET3 ESMF_PIO                    : enabled
20220714 020811.282 INFO             PET3 ESMF_YAMLCPP                : enabled
20220714 020811.283 INFO             PET3  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20220714 020811.283 INFO             PET3  NUMBER_OF_PROCESSORS           4
20220714 020811.285 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.287 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.287 INFO             PET3  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20220714 020811.289 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.289 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.290 INFO             PET3  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20220714 020811.292 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.294 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.294 INFO             PET3  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20220714 020811.307 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.319 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.319 INFO             PET3  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20220714 020811.331 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.345 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.346 INFO             PET3  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20220714 020811.350 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20220714 020811.351 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20220714 020811.352 INFO             PET3  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20220714 020811.352 INFO             PET3  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20220714 020811.352 INFO             PET3 Finalizing ESMF
