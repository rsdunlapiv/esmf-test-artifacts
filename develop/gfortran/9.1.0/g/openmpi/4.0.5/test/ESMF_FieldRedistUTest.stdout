2022-07-14 01:49:41
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  PET           1  Test Elapsed Time    72.692000000000007       msec.
  PET           3  Test Elapsed Time    72.920000000000016       msec.
  PET           2  Test Elapsed Time    72.650999999999996       msec.
  PET           0  Test Elapsed Time    72.919999999999987       msec.
